ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/ImportOrderingRule.kt

```diff
@@ -23,7 +24,7 @@ class ImportOrderingRule : Rule("import-ordering") {
             if (children.isNotEmpty()) {
                 val imports = children.filter { it.elementType == IMPORT_DIRECTIVE }
                 val sortedImports = imports.sortedBy { it.text }
-                if (imports != sortedImports) {
+                if (imports != sortedImports || hasTooMuchWhitespace(children)) {
                     emit(node.startOffset, "Imports must be ordered in lexicographic order", true)
```

Comment:
Could we also move this entire rule under the experimental ruleset? Can also be done as a separate PR.

Reply:
I will do that as soon as this one is merged.

Reply:
sounds good.